src/daemon-commander.ts

```diff
@@ -19,8 +19,10 @@
 import got from 'got';
 import { isWindows } from './util';
 
+export type CrcStatus = 'Running' | 'Starting' | 'Stopping' | 'Stopped' | 'No Cluster' | 'Error' | 'Unknown';
```

Comment:
'No cluster' is new... extension specific?

Reply:
Yes